src/util/auth/google_authentication_repository.ts

```diff
@@ -0,0 +1,106 @@
+import { secureStore } from "@storage";
+import { get } from "svelte/store";
+
+const storedAccessToken = secureStore<string>("access_token", null);
+const storedRefreshToken = secureStore<string>("refresh_token", null);
+
+// Scopes must match the scopes configured in the Google Developers Console.
+const SCOPES = [
+  "https://www.googleapis.com/auth/photoslibrary.readonly",
+].join("_");
+
+interface AuthenticationResponse {
+  accessToken: string,
+  refreshToken: string
+};
+
+const save = (accessToken: string, refreshToken: string) => {
+  console.log(`Saving access_token:(${accessToken}), refresh_token:(${refreshToken})`);
```

Comment:
If this needs to be encrypted, we should not print it either no?

Reply:
nope we shouldnt

Reply:
it was more for debug purposes, we should get something that prints the logs based on the build type

Reply:
we can create a logger function that checks `import.meta.env.MODE` and only outputs to the console when the mode is development, but that's not in the scope here i guess